src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
+  }
+
+  md-tab-item {
+    padding-top: $toolbar-height-size-base / 2;
+    text-transform: none;
```

Comment:
I see. I think we should be going with standard angular material components, unless it gives benefits for users to change them. With this we'll save engineering resources to implement features, rather than play with frameworks. Does this make sense?